embedx/config.schema.json

```diff
@@ -819,6 +819,9 @@
         "webauthn": {
           "$ref": "#/definitions/selfServiceAfterSettingsAuthMethod"
         },
+        "passkey": {
```

Comment:
@aeneasr passkey or passkeys? your call.

Reply:
PassKeys is the official name so I would stick to that :)